src/spring-bceid-starter/src/main/java/ca/bc/gov/open/bceid/starter/AutoConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package ca.bc.gov.open.bceid.starter;
+
+import ca.bc.gov.open.bceid.starter.account.BCeIDAccountService;
+import ca.bc.gov.open.bceid.starter.account.BCeIDAccountServiceImpl;
+import ca.bc.gov.open.bceid.starter.account.mappers.AccountDetailRequestMapper;
+import ca.bc.gov.open.bceid.starter.account.mappers.AccountDetailRequestMapperImpl;
+import ca.bc.gov.open.bceid.starter.account.mappers.IndividualIdentityMapper;
+import ca.bc.gov.open.bceid.starter.account.mappers.IndividualIdentityMapperImpl;
+import ca.bceid.webservices.client.v9.BCeIDServiceSoap;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.cxf.jaxws.JaxWsProxyFactoryBean;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties(BCeIdProperties.class)
+public class AutoConfiguration {
+
+    private final BCeIdProperties bCeIdProperties;
+
+    public AutoConfiguration(BCeIdProperties bCeIdProperties) {
+        this.bCeIdProperties = bCeIdProperties;
```

Comment:
question:
why not just `bceidProperties`? seems like weird casing with `bCeId`

Reply:
good call however all the services are BCeID, so I kept it consistent...